api/routes/users/post.js

```diff
@@ -9,21 +9,23 @@ module.exports = (app, UserModel = defaultUserModel) => {
     logger.silly(req, `attempting to create new user [${req.body.email}]`);
     if (req.body.email && req.body.password) {
       try {
-
         const posted = {
           email: req.body.email,
           password: req.body.password
         };
-          
+
         // If these values are not passed in, don't set them at all, even
         // to undefined.  Undefined will make the data model explode.
         if (req.body.name) {
           posted.name = req.body.name;
         }
-        if(req.body.state) {
+        if (req.body.role) {
+          posted.auth_role = req.body.role;
+        }
```

Comment:
Allow the authorization role to be set when the user account is created.  Previously the only way to set the role was to edit the account after it was created, which seems silly.  Jen's mockups included setting the role when creating the account, so I went ahead and added it.